frontend/flaskserver.py

```diff
@@ -80,10 +80,22 @@ def get_mime_string(path):
     secure = True
 
 # SSL context
-context = ('selfsigned.crt', 'decserver.key')
+context = (exec_dir+'/selfsigned.crt', exec_dir+'/decserver.key')
 
 # ROUTES ###############################################################
 
+@app.after_request
+def add_header(r):
+    """
+    Add headers to both force latest IE rendering engine or Chrome Frame,
+    and also to cache the rendered page for 10 minutes.
+    """
+    r.headers["Cache-Control"] = "no-cache, no-store, must-revalidate"
+    r.headers["Pragma"] = "no-cache"
+    r.headers["Expires"] = "0"
```

Comment:
I like this bit; however, shouldn't Expires be set to 10 (or 600) for 10 minutes to expire, not 0? Or is 10 mins some standard default?